src/main/resources/assets/js/components/modals/AboutModalComponent.jsx

```diff
@@ -37,9 +29,19 @@ var AboutModalComponent = React.createClass({
     return this.state.info;
   },
 
+  getInfo: function (attr, showUnspecifiedText) {
```

Comment:
Why do we need `showUnspecifiedText` as a special option? Seems like a good default case for the version and then we get rid of a ton of special logic.